client/ayon_core/hosts/houdini/api/lib.py

```diff
@@ -811,6 +811,48 @@ def get_current_context_template_data_with_folder_attrs():
     return template_data
 
 
+def set_review_color_space(opengl_node, review_color_space="", log=None):
+    """Set ociocolorspace parameter for the given OpenGL node.
+
+    Set `ociocolorspace` parameter of the given OpenGl node
+    to to the given review_color_space value.
+    If review_color_space is empty, a default colorspace corresponding to
+    the display & view of the current Houdini session will be used.
+
+    Args:
+        opengl_node (hou.Node): ROP node to set its ociocolorspace parm.
+        review_color_space (str): Colorspace value for ociocolorspace parm.
+        log (logging.Logger): Logger to log to.
+    """
+
+    if log is None:
+        log = self.log
```

Comment:
is there any reason for this? Down the line `self.log` is used nevertheless and `log` in this case will be always `None`

Reply:
Is this function even used anywhere? :) And if not, should it?

Reply:
> Is this function even used anywhere? :) And if not, should it?

I feel so stupid. I forgot to use it. 
added here: https://github.com/ynput/ayon-core/pull/36/commits/e8e1a1a2dffd197e61101ccc910c9d3f69c01a1f